shiprush.php

```diff
@@ -109,35 +113,127 @@ function __construct() {
 			function shiprush_button() {
 				add_meta_box( 'woocommerce-shiprush', __('ShipRush', 'WC_ShipRush'), array( &$this, 'shiprush_meta_box' ), 'shop_order', 'normal', 'high');
 			}
+		
+			/**
+			 * Add ShipRush support for WooCommerce Tracking plugin.
+			 *
+			 * @access public
+			 */
+			function register_shiprush_tracking( $providers ) {
+				$providers['United States']['Ship Rush'] = 'https://tools.usps.com/go/TrackConfirmAction_input?strOrigTrackNum=%1$s';
```

Comment:
Sean are you sure it is only for USPS?

Reply:
Not so sure now that you said it.
Gonna check their docs again. Pretty sure that I had a hard time finding this one.

Reply:
Is this required? can't we only display the tracking code? and we could see how to improve this in the future